automation_tools/__init__.py

```diff
@@ -1622,6 +1623,30 @@ def setenforce(mode):
     run('setenforce {0}'.format(mode))
 
 
+def host_pings(host):
```

Comment:
> I am seriously not understanding how do I...

I think you missed my point. If ping doesn't provide you with requisite functionality, then don't sit there and bang your head against the wall. I'm simply asking you to read the manual before writing any code.